package.json

```diff
@@ -50,15 +50,15 @@
     "kleur": "^4.1.5",
     "micromatch": "^4.0.5",
     "pirates": "^4.0.5",
-    "playwright-chromium": "=1.31.1",
-    "playwright-core": "=1.31.1",
+    "playwright-chromium": "=1.32.3",
+    "playwright-core": "=1.32.3",
     "semver": "^7.3.8",
     "sharp": "^0.31.1",
     "snakecase-keys": "^4.0.1",
     "sonic-boom": "^3.2.0",
     "source-map-support": "^0.5.21",
     "typescript": "^4.8.4",
-    "undici": "^5.11.0",
+    "undici": "^5.21.2",
```

Comment:
Not sure why its not picking up the changes from the last commit in `main`. can you do a merge?